### DISCUSSION TITLE
My account was marked as spam, where can I get support to get me unmarked as spam?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#80307] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi, 

My account has been marked as spam.  Likely I think because I attempted to use a disposable email address originally when I signed up for GitHub.  That was a mistake, but I did not realise that my account was marked as spam until today, when i was attempting to do something and couldnt (upgrade).

I have now changed my email addresses to be proper email addresses and not disposable ones, I have also set up 2FA on my account.

Where can I get help, to get my account unmarked as spam?

Thanks

Jason (Jay).

**Comment by Ableez** [comment]
been facing the same issue, i had to post a support ticket "Reinstatement Request #2559099", it has been over a week no reply from github on how to sort this out. its really annoying. cause i can't even do anything.
